app/src/renderer/vuex/modules/wallet.js

```diff
@@ -1,5 +1,6 @@
 let fs = require("fs-extra")
 let { join } = require("path")
+let { uniqBy } = require("lodash")
```

Comment:
Please use `const` whenever possible.

Reply:
IMO we should talk about our style for this. I don't think using `const` is usually worth it since I don't think it's ever prevented a bug for me.

It actually doesn't even error when re-assigned outside of strict mode, which seems like it could even hide some bugs:
```js
let foo = 5
foo = 6
console.log(foo) // => 5
```
So maybe it makes sense if we also make a rule of enabling strict mode for each file, but I'm not convinced going through and changing all that is worth it.

Reply:
Let's not use const everywhere then. We can discuss in our team meeting tomorrow if we want to enforce this.